public/app/dashboard/profile.css

```diff
@@ -43,12 +43,27 @@
 }
 
 .profile-engagement-column {
-  border-radius: 4px;
+  border-radius: 5px;
   border: 1px solid lightgrey;
-  margin: 5px;
   padding: 10px 10px 10px 10px;
 }
 
+.profile-engagement-column h2 {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 1.3;
+  word-wrap: break-word;
+}
```

Comment:
Despite decreased font-size, I keep `h2` for appropriate structure (accessibility).